GPyOpt/__version__.py

```diff
@@ -1 +1,6 @@
-__version__ = "1.2.1"
+from pkg_resources import get_distribution, DistributionNotFound
```

Comment:
I am not familiar with this package, so a naive question maybe: is it always guaranteed to be available? Does it come with a standard Python installation, or perhaps we need to declare a new dependency for it?

Reply:
almost standard :)

it comes with setuptools installation, which is not in the clean python installation, but most of python tutorials recommend you to install it.

I think you cannot use pip, nor setup.py without a setuptools installed :)